frontend/src/component/admin/apiToken/ProjectsList/ProjectsList.tsx

```diff
@@ -1,11 +1,13 @@
+import { Fragment, type FC } from 'react';
 import { styled } from '@mui/material';
 import { Highlighter } from 'component/common/Highlighter/Highlighter';
 import { HtmlTooltip } from 'component/common/HtmlTooltip/HtmlTooltip';
 import { LinkCell } from 'component/common/Table/cells/LinkCell/LinkCell';
 import { TextCell } from 'component/common/Table/cells/TextCell/TextCell';
 import { useSearchHighlightContext } from 'component/common/Table/SearchHighlightContext/SearchHighlightContext';
-import { Fragment, type FC } from 'react';
+import ErrorIcon from '@mui/icons-material/Error';
```

Comment:
Is an error icon the right icon to show here? Wouldn't a warning be more appropriate? 🤔

Reply:
I don't want this to be ignored. Tokens like this, in theory, leak data.

Reply:
Fair enough. There's nothing preventing you from just ignoring it (for now), though. But I'm easy either way; do whatever you feel is best ☺️